pkg/register/document.go

```diff
@@ -69,6 +69,30 @@ type RegisterDocument struct {
 	Metadata               Metadata                  `json:"metadata,omitempty"`
 }
 
+func (key RegisterPublicKey) ExistsInSlice(cmp []RegisterPublicKey) error {
```

Comment:
does this naming make sense? it's not just checking existence, it's performing validation of revocation and removal?

Reply:
689aee0f763917eb0ca06eda79d716359a98ac44